### PR TITLE
Added assets folder and ERD diagram to assets folder

### PR DESCRIPTION
**Pull request by CalebMathers**
For issue #98 
For issue #91 

Created an assets directory and added the ERD diagram

### CODE REVIEW

**Review by AlexOBarnes — CHANGES_REQUESTED** [review]
We should include short name for constellation
Otherwise LGTM

**Review by AlexOBarnes — APPROVED** [review]
thanks for making those changes

File: assets/ERD_starwatch.png
Comment (olliejth) [inline assets/ERD_starwatch.png:1]: Is image worth storing historically?

**Review by olliejth — APPROVED** [review]
LGTM - Will have to decide on small design choices for querying later but otherwise fine.

**Pull request merged**